EmerisHQ/demeris
# fix: fixed connecting wallet and disconnecting wallet displays different airdrop state

## Description
Please refer to issue #1589 . The airdrops state work perfectly now, when the user connects wallet, the eligibility check is performed and applied to the airdrops table properly. Also the reverse happens for when the user disconnects wallet.


## Feature flags
Please use this feature flag ?VITE_FEATURE_AIRDROPS_FEATURE=1 to test this

## Testing

1. Add feature flag to url
2. Go to airdrops page
3. Connect wallet and see eligibility stasuses appended to airdrops table
4. Disconnect wallet and see eligibility stasuses removed from airdrops table

https://www.loom.com/share/5e734ef9eabe4f4ba0b254c2a2abc35a
